MELA/interface/ZZMatrixElement.h

```diff
@@ -20,7 +20,8 @@ class ZZMatrixElement{
   ~ZZMatrixElement();
 
   void computeXS(
-    float &mevalue
+    float &mevalue,
+    int nhel=-1
```

Comment:
Again, why this default?

Reply:
See previous answer about inputs to the `smatrixhel` function.